keyboards/melgeek/z70ultra/rev1/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
```

Comment:
And same for the rules.mk - just need to add `DEFAULT_FOLDER = rev1`.

Reply:
```
- DEFAULT_FOLDER = rev1
+ DEFAULT_FOLDER = melgeek/z70ultra/rev1
```